src/tarkov-data-manager/modules/image-create.js

```diff
@@ -0,0 +1,79 @@
+const fs = require('fs');
+const Jimp = require('jimp-compact');
```

Comment:
Is use of `jimp-compact` temporary? We seen that `sharp` is much faster in image processing for `tarkov-dev`

Reply:
for just _resiizing_, sharp is much faster. However, I tried converting to sharp for the image generator and it just isn't as good. It lacks many features of jimp and is all-around buggy.